src/shared/ui/MainLogoIcon/index.tsx

```diff
@@ -0,0 +1,2 @@
+import MainLogoIcon from "./MainLogoIcon";
+export default MainLogoIcon;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.